fixtures/schema.js

```diff
@@ -163,6 +169,7 @@ const Manufacturers = schema({
     'name': NonEmptyString,
     '?comment': NonEmptyMultiLineString,
     '?website': URL,
+    '?rdmId': Number.min(0).step(1),
```

Comment:
Max is defined for this 0xffff it should be (I'll double check the standard later), with a few exceptions it can't be.